test/e2e/framework/helper/featureset/featureset.go

```diff
@@ -131,4 +153,8 @@ const (
 	// IssueCAFeature denotes whether the target issuer is able to issue CA
 	// certificates (i.e., certificates for which the CA basicConstraint is true)
 	IssueCAFeature Feature = "IssueCA"
+
+	// LongDomainFeatureSet denotes whether the target issuer is able to sign
+	// a certificate that defines a long domain
+	LongDomainFeatureSet Feature = "LongDomain"
```

Comment:
What is a LongDomain in this context? (i.e. how long?) - I'd have assumed all our issuers should be able to sign for any domain that is within the length restrictions noted in RFCs.